[DOCS-1009] Add how to configure team-invite behavior more places (#1135)
wandb/docs: content/guides/models/app/settings-page/team-settings.md

## Privacy

Navigate to the **Privacy** section to change privacy settings. Only organization admins can modify privacy setting.

> - Optionally disable the ability to make future projects public or to share reports publicly.

Reviewer: ```suggestion
- Turn off the ability to make future projects public or to share reports publicly.
```

Also, this sentence is a little confusing. Maybe consider?

"Optionally make future projects or reports public."

Author: I think you're reading it backward, this is about turning that off. I'll edit your edit.

> - Optionally allow any team member to invite other members, rather than only team admins.

Author: ```suggestion
- Allow any team member to invite other members, rather than only team admins.
```

> - Enable or disable code saving by default.

Reviewer: ```suggestion
- Manage whether code saving is turned on by default.
```
